src/components/Callout/index.js

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {
+  H3,
+  H2,
+  H1,
+  PrimaryButton,
+  ButtonText,
+  GREEN,
+} from "../../styles/styles";
+import { Container } from "./styled";
+
+const CallOut = ({ label, header }) => (
+  <>
+    <Container>
+      <H2>{header}</H2>
+      <PrimaryButton color={GREEN}>
+        <ButtonText>
+          <H1>{label}</H1>
+        </ButtonText>
```

Comment:
this is a `styled.h3` so make sure not to have `H1` as a child (this would be fixed by using buttons from mackenzies PR if you just want to pull that in when she merges)